aten/src/ATen/native/TensorShape.cpp

```diff
@@ -2130,7 +2130,7 @@ inline Tensor view_impl(const Tensor& self, IntArrayRef size) {
   TORCH_CHECK(stride.has_value(), "view size is "
     "not compatible with input tensor's size and stride (at least one dimension"
     " spans across two contiguous subspaces). Use .reshape(...) instead.");
-  return alias_with_sizes_and_strides(self, inferred_size, *stride);
+  return self.as_strided(inferred_size, *stride);
```

Comment:
The whole point of `unsafe_view` is to be faster than doing a proper view that goes through the dispatcher.
So if we do this, we might as well remove `unsafe_view` altogether.

Reply:
oh haha. Then maybe the `CompositeExplicitAutograd` label is just false advertising.

Ok, I'm actually going to just remove this PR. I guess we could replace `CompositeExplicitAutograd` with `CPU, CUDA, QuantizedCPU, QuantizedCUDA`, etc, but we risk breaking 3rd party backends that happened to be relying on it.

I have another PR later that will add a new alias key for basically "all backends except functional backends", which might be a good candidate for this kernel (since functional backends like XLA/LTC won't like the `alias_with_sizes_and_strides` call).